### ISSUE TITLE
How do we build a shared culture around accessibility?

### ISSUE THREAD

**Issue by sorensensig** [digdir/user-experience#38]
A lack of focus on accessibility today might be due to lack of knowledge. It could be interesting to conduct a study to understand the knowledge-level of accessibility across all product teams.

Accessibility-bingo to reward those who do well on accessibility and clearly communicate to the rest of the company the effort of those who do well.

How can accessibility be built into our workflows?
